lib/wallet/src/Cardano/Wallet/Write/Tx.hs

```diff
@@ -752,6 +771,46 @@ fromCardanoUTxO = withStandardCryptoConstraint (recentEra @era) $
   where
     unCardanoUTxO (Cardano.UTxO m) = m
 
+cardanoValueFromCoreValue
```

Comment:
```suggestion
toCardanoValue
```

Reply:
Fixed in 32a1fb80fc8aac4758720bdcebdc733ae7904b21.